scipy/linalg/lapack.py

```diff
@@ -482,7 +482,7 @@ def get_lapack_funcs(names, arrays=(), dtype=None):
         used, otherwise the most generic type in arrays will be used.
 
     dtype : str or dtype, optional
-        Data-type specifier. Not used if `arrays` is non-empty.
+        Data-type specifier. Not used if ``arrays`` is non-empty.
```

Comment:
I've checked the document and it is shown as italics. Don't know if this is the regular behavior but I guess monospaced font is meant here is that right?